capa/main.py

```diff
@@ -446,7 +446,9 @@ def main(argv=None):
     parser = argparse.ArgumentParser(
         description=desc, epilog=epilog, formatter_class=argparse.RawDescriptionHelpFormatter
     )
-    parser.add_argument("sample", type=str, help="path to sample to analyze")
+    parser.add_argument(
+        "sample", type=lambda s: s.decode(sys.getfilesystemencoding()), help="path to sample to analyze"
```

Comment:
can you also include an explanation here for why this is needed? i presume that argparse doesn't handle unicode well, so including some details on what you found will help future archeologists.